common/scripts/docker-build.sh

```diff
@@ -188,7 +188,8 @@ popd > /dev/null 2>&1
 
 # validate docker version against minimum required docker version
 # docker_version=$(docker version --format '{{.Server.Version}}')
-docker_version=$(docker version | grep 'Version:' | awk '{print $2}')
+docker_version=$(docker version)
+docker_version=$(echo "$docker_version" | grep 'Version:' | awk '{print $2}')
```

Comment:
@ravihansa3000 This was one way of making "docker version" fail if the daemon was not running. If docker daemon was available, it will simply store the result on $docker_version variable which we later use to parse the version number. I could change the variable names, if that makes it easier to understand.